### ISSUE TITLE
1080 crash

### ISSUE THREAD

**Issue by Gbyzzz** [OhGodACompany/OhGodAnETHlargementPill#20]
Hi,
Have 2x1080ti(MSI and Palit) - working fine, but I can't  make it work with Zotac 1080 AMP(BIOS 86.04.17.00.A2) - crashing(system restarts) all the time. Tried --revA - same issue. Miner - Claymore 11.7.
Is there any cure or I just need to wait for improved pill?

**Comment by OhGodABird** [comment]
Please report your settings, it's helpful for understanding issues like this better.

Your other cards seem to be heavily overclocked, therefore for now I am assuming you are overclocking your 1080 too. In that case, start with the stock clocks and a reasonable power limit instead of starting with clocks that "should work" according to you.

I also suggest verifying the hashrate on the pool for the other cards. Too high clocks can cause deceivingly high hashrates while actually producing a lot of invalid solutions, therefore reducing the valid solutions submitted to the pool.

**Comment by Drift333** [comment]
Hi!
I have the same problem.
As soon as I run OhGodAnETHlargementPill, the system reboots. WINDOWS 10 reboots without overclocking and Mining.
The drivers are the last. In the system 1 palit geforce gtx 1080 dual, 4 gigabytes 1080 and three Gainward 1070.
Us other rigs with 1080TI everything works perfectly !!!

**Comment by OhGodABird** [comment]
@Drift333 Have you tried running the pill with --revA as described in the prescription? With 8 cards and such a severe reaction I recommend running it like `OhGodAnETHlargementPill-r2.exe --revA 0,1,2,3,4,5,6` first. If that solves the issue, proceed to remove the indexes of 1070s which shouldn't make any difference, then try to remove the indexes of 1080s one by one, testing mining at each step.

**Comment by Gbyzzz** [comment]
@OhGodABird yes, it was overclocking. Interesting that without pills it was working fine. I have another question, command "--revA 0" for example means that pills will not be used on GPU0, right?

**Comment by Drift333** [comment]
I tried the command --revA 0,1,2,3,4,5,6,7 and really the tablet started. But when I try to remove any card, including 1070 and run the miner system, it goes into reboot. Maybe I have 1080 old memory on all the cards?

**Comment by OhGodABird** [comment]
@Gbyzzz 
The prescription covers revA too, but I'll explain it here a bit more clearly. The pill has two "solutions": revA and revB. The default is revB, but older GPUs (which tend to be 1080s) only take the revA well, that why it's needed to be specified. There is currently no option to exclude a GPU, because the amount of GPUs not taking at least the revA "solution" is expected to range from none to an insignificant number.

You also have to keep in mind, that your GPUs are working harder for a higher hashrate, so clocks that were borderline sufficient for operation without the pill can be expected to be too high while taking the pill.

@Drift333
If removing an index results in crashing, then in your case it means that the GPU associated with that index was an old 1080 that needs revA. If you are relying on your miner for indexes, do note that it may use a different order for various reasons, so trying to remove every index one by one is your best bet. Try `--revA 1,2,3,4,5,6,7`, if it's not working, then move on to `--revA 0,2,3,4,5,6,7`, if that's fine then keep that index removed and move to the next like `--revA 0,3,4,5,6,7` and so on.

**Comment by Drift333** [comment]
I made everything as described. At any hit on the card 1080 at the start of mining system goes into reboot. Works only 0,1,2,3,4,5,6,7 or 0,2,3,4,7 if all five 1080 graphics cards are turned off. If I make a change at least one digit, the system goes into a reboot.

**Comment by OhGodABird** [comment]
@Drift333 
As it's already described in multiple places, --revA doesn't turn off anything, it just uses a solution suitable for older GPUs. If you have to apply it for all of your 1080s, it just means that all of them are older models, and simply that's the best they can do.

**Comment by Gbyzzz** [comment]
@OhGodABird hmm, ok, I have another question then. With revA 1080 gives 25-26 Mh/s. Is it ok?

**Comment by OhGodABird** [comment]
@Gbyzzz 
Seems to be a bit low, but could be reasonable depending on your settings. Check your core clock to ensure it's not throttled to a low value, and if you are not overclocking your memory, I also suggest trying that. Do note that by default, the highest memory clock state isn't applied when mining, so assuming you don't have that modified, you are likely below the memory clock the GPU was advertised with. After finding a satisfying set of settings, keep an eye on the miner and / or the pool for a while to make sure the GPU isn't clocked too high, which can result both in crashing and having lower effective hashrate by producing less valid solutions.

**Comment by extremenelson** [comment]
I also have a Zotac 1080ti card but am running Arch Linux with the 4.16.1-1-zen kernel and the NVIDIA 390-48-10 drivers. I get a segmentation fault when I try and run the pill under sudo. The last several lines of running under strace are:

mprotect(0x7fa48d13e000, 131072, PROT_READ|PROT_WRITE) = 0
mprotect(0x7fa48d026000, 1048576, PROT_READ) = 0
rt_sigreturn({mask=[]})                 = 16532
clone(child_stack=0x7fa48c197fb0, flags=CLONE_VM|CLONE_FS|CLONE_FILES|CLONE_SIGHAND|CLONE_THREAD|CLONE_SYSVSEM|CLONE_SETTLS|CLONE_PARENT_SETTID|CLONE_CHILD_CLEARTID, parent_tidptr=0x7fa48c1989d0, tls=0x7fa48c198700, child_tidptr=0x7fa48c1989d0) = 8239
futex(0x7fa48c1989d0, FUTEX_WAIT, 8239, NULL) = ?
+++ killed by SIGSEGV (core dumped) +++

Would love to get this running on my rig. If there is any other info you need, I would be happy to provide it.

**Comment by mrfhenix** [comment]
Dear Doc

tried both remedies with my 6 1080,s (with and without OC) latest Nvidia Driver 391.35
--Rev A solution and No Rev

* tried Default MSAfterBurner Reset (with and without RevA)
* tried UnderVoltage
* under clocked and OC

*Aorus Gaming 1080 Latest Nvios Revition
*windows 10 (latest update)

BTW: my second Rig is feeling younger than ever
Worked as Magic.. 6 msi Gaming X's 1080
Same configuration both rigs are CLONES just Different Gpus

Do i need to send you any Aditional Studies?? DAta Logs??

**Comment by nkyms** [comment]
I've tried the same configurations as mrfhenix has posted above.
I get crashes without the revA fix and with it doesn't have any effect on hash.
Have tried versions 10, 11.6, 11.7 of Claymore Miner, and older and newer versions of nVidia drivers.

**Comment by FatShady2017** [comment]
I'm noob. How do you actually run with --revA? like where do I put this command?

**Comment by kolet** [comment]
@OhGodABird 
i've read what u said, i have 8 1080 FE PNY, i guess they are old version since runing the .exe as admin crashes the pc.
so i followed ur instruction and did --revA 0,1,2,3,4,5,6,7 , thing is i am not getting near 50, or 40, i receive more like 25~ after oc 300mhz on the memory, is that normal? thanks :) (FOR EVERYTHING! ^,^)

**Comment by CryptoMang** [comment]
It's just odd to me that I have 3 rigs of 9 1080's that not a single card will work with the Pill.  I have about 6 other 1080 rigs of 9 that work without issue.  All of my setups are identical.  I get all kinds of "warp exceptions" if I try to use the pill on those 3 rigs that aren't happy.  Can someone help me make my rigs happy?

**Comment by 0sm3l** [comment]
Same problem for me with a Zotac 1080

**Comment by diemcizone** [comment]
@CryptoMang the same condition, 4 pieces rig works. 32 GTX 1080 Gaming G1 with Gigabyte @ + 1000 Memory and 37.6MH / s.
2 pieces of rig do not work, with the same installation. Where is the problem? settings and configuration are the same.

**Comment by CryptoMang** [comment]
For some reason it is on my rigs that can't run the Pill.  The ones that can run the Pill have a much lower clock.  My suspicion is that I cannot run the Pill because of this, but I'm doing more research.  What OS are you guys using?  I'm using SMOS on Linux.

**Comment by MrMineitall** [comment]
Instead of opening a new thread ill write my issue here:

I decided to mine eth with my 1080s and 1080ti... the 1080ti didnt need the revA so all are working like a charm on hiveOS

Now, the 1080 rig, after playing with the revA command, i found out that i have 2 cards (which i havent identified which cards are physically) need the revA command where the other cards run without it. All good apart that the cards that i had to use the revA command (--revA 6,9) (lol) they dont get to 33mh/s as the others that DONT need the revA do. The hashrate is exactly the same as if the ethenlargment was turned OFF. Is there a way or another command i need to add in order for this 2 cards to go up to 33mh/s as the others?

Thank you very much for this tool and happy mining to you all.

Regards,
T

**Comment by mmontales** [comment]
supper noob here... please tell me where to run this command? OhGodAnETHlargementPill-r2.exe --revA 0,1,2 ... correct me if im wrong, i'll open the cmd and type the OhGodAnETHlargementPill-r2.exe --revA 0,1,2? is that correct?.. i have 2 gtx 1080 and it's crashing after a while.
